sale_import_base/tests/common_sale_order_import.py

```diff
@@ -0,0 +1,183 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.addons.datamodel.tests.common import SavepointDatamodelCase
+from odoo.addons.sale.tests.test_sale_common import TestCommonSaleNoChart
+
+
+class SaleImportCase(SavepointDatamodelCase, TestCommonSaleNoChart):
+    @classmethod
+    def setUpClass(cls):
+        super(SaleImportCase, cls).setUpClass()
+        cls.setUpClassicProducts()
+        cls.setUpAddresses()
+        cls.setUpLines()
+        cls.setUpImport()
+        cls.setUpTaxes()
+        cls.setUpFpos()
+
+    @classmethod
+    def setUpAddresses(cls):
+        cls.partner_thomasjean = cls.env["res.partner"].create(
+            {
+                "name": "Thomas Jean",
+                "email": "thomasjean@gmail.com",
+                "street": "initial address",
+                "street2": "initial address2",
+                "zip": "0",
+                "city": "initial city",
+                "country_id": cls.env.ref("base.fr").id,
+            }
+        )
+        cls.addr_customer_example = {
+            "name": "Thomas Jean",
+            "street": "1 rue de Jean",
+            "street2": "bis",
+            "zip": "69100",
+            # 'state_code': 'CR-SJ',
+            "city": "Lyon",
+            "email": "thomasjean@gmail.com",
+            "country_code": "FR",
+            "external_id": "EBAY_CUST123",
```

Comment:
puts thomas_jean as external key, it will be more understandable then EBAY_CUST123 in your test latter